Makefile

```diff
@@ -21,10 +21,6 @@ explorer:
 	mkdir -p bin/
 	echo "Creating frontend asset bundle..."
 	go run cmd/bundle/main.go
-	echo "Generating swagger docs..."
```

Comment:
Question: Don't we need a new build process via the redocly cli?

Reply:
right, I adapted the build pipeline & the Dockerfile to automatically generate the docs bundle